qiskit_experiments/calibration_management/calibrations.py

```diff
@@ -52,18 +56,52 @@ class Calibrations:
     ScheduleBlock are supported.
     """
 
+    # The name of the parameter under which the qubit frequencies are registered.
+    __qubit_freq_parameter__ = "qubit_lo_freq"
+
+    # The name of the parameter under which the readout frequencies are registered.
+    __readout_freq_parameter__ = "meas_lo_freq"
+
     # The channel indices need to be parameterized following this regex.
     __channel_pattern__ = r"^ch\d[\.\d]*\${0,1}[\d]*$"
 
-    def __init__(self, control_config: Dict[Tuple[int, ...], List[ControlChannel]] = None):
+    def __init__(
+        self,
+        coupling_map: Optional[List[List[int]]] = None,
+        control_config: Optional[Dict[Tuple[int, ...], List[ControlChannel]]] = None,
```

Comment:
```suggestion
        control_channel_map: Optional[Dict[Tuple[int, ...], List[ControlChannel]]] = None,
```
or control channel assignment? control config sounds like something different.

Reply:
Sounds good. Done here: 2d6d8ef